cmd/commands/operator.go

```diff
@@ -45,6 +46,7 @@ var setCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
 		clientsets := GetClientsets(cmd.Context())
+		CheckOperatorPod(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
```

Comment:
Why does the operator pod need to be running? Aren't we just updating the configmap?

Reply:
we are making a change in configmap `rook-operator...` and if the pod is not running there will be no effect right? To update in values in the configmap operator pod should be running otherwise values will be updated but operator will not reconcile those

Reply:
Correct, the operator wouldn't reconcile them if it's down. I can imagine someone would want to update a setting and then starting the operator separately, for example to see different log levels at startup.

Reply:
okay, I have removed the check